future-apps/payroll/contracts/Payroll.sol

```diff
@@ -39,7 +39,6 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
     bytes32 constant public MODIFY_PRICE_FEED_ROLE = 0x74350efbcba8b85341c5bbf70cc34e2a585fc1463524773a12fa0a71d4eb9302;
     bytes32 constant public MODIFY_RATE_EXPIRY_ROLE = 0x79fe989a8899060dfbdabb174ebb96616fa9f1d9dadd739f8d814cbab452404e;
 
-    uint128 internal constant ONE = 10 ** 18; // 10^18 is considered 1 in the price feed to allow for decimal calculations
```

Comment:
It's quite weird the changes in #902 are showing up in this diff :/

Reply:
It doesn't show in the commit history though :shrug:

e52aae4bcae478633e57e4106bcbd136f0ad3441

Reply:
weird :/